app/models/aliquot.rb

```diff
@@ -59,4 +59,43 @@ def tagged?
   def collection?
     false
   end
+
+  # Returns the id to publish
```

Comment:
It would feel better to have all the published data together in one object as it would all be called together.

Reply:
Totally agree, and I really wanted to do it that way and started of  with a 'publishing_data ' method in aliquot. I tried it, but didn't work  with how the DatastructureBuilder is written. To my understanding this requires changes in the current DataStructureBuilder class where a '.' access with a 'model' type recursively send it to the object itself.